images/win/scripts/Installers/Install-SQLPowerShellTools.ps1

```diff
@@ -1,21 +0,0 @@
-################################################################################
-##  File:  Install-SQLPowerShellTools.ps1
-##  Desc:  Install SQL PowerShell tool
-################################################################################
-
-Import-Module -Name ImageHelpers -Force
-
-$BaseUrl = "https://download.microsoft.com/download/8/7/2/872BCECA-C849-4B40-8EBE-21D48CDF1456/ENU/x64"
-
-# install required MSIs
```

Comment:
I don't think that is the right approach to remove module without any ideas who uses it. What's happened with customer software if he needs to use older version of module?

Reply:
@maxim-lobanov @AlenaSviridenko could you please provide your thoughts about sqlps module removal?

Reply:
@LeonidLapshin Agree with Aleks,
Why would you like to remove this module?

Reply:
The module is actually stale and it's functionality is duplicated in SqlServer module so I guess there is no need for it. 
But I agree that it can be currently in use so may be it's not the best choice to remove it within this PR without announcement at least.